modules/jfx.incubator.richtext/src/main/java/jfx/incubator/scene/control/richtext/RichTextArea.java

```diff
@@ -385,34 +385,22 @@ public final TextPos getAnchorPosition() {
     }
 
     /**
-     * Determines the caret blink period.  This property cannot be set to {@code null}.
+     * Determines the caret blink period.  A {@code null} value will be interpreted as 1000 ms.
      * <p>
      * This property can be styled with CSS using {@code -fx-caret-blink-period} name.
      * @implNote The property object implements {@link StyleableProperty} interface.
      *
      * @return the caret blink period property
-     * @defaultValue 1000 ms
+     * @defaultValue null
```

Comment:
I am not in favor of this change. From a user API perspective, the default should be 1000 ms, regardless of whether or not null should be also treated as valid, and treated as 1000 ms.

Separately, I'm not sure I see the need for allowing null, but there might be a reason I'm missing.